backend/app/services/adapters/executor_job.py

```
@@ -74,118 +89,291 @@ def cleanup_stale_executors(self, db: Session) -> None:
+                Subtask.status.in_(
+                    [
+                        SubtaskStatus.PENDING,
+                        SubtaskStatus.COMPLETED,
+                        SubtaskStatus.FAILED,
+                        SubtaskStatus.CANCELLED,
+                    ]
+                ),
+                Subtask.updated_at <= cutoff,
+                Subtask.executor_name.isnot(None),
+                Subtask.executor_name != "",
+                Subtask.executor_deleted_at == False,
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `False`; use `not Subtask.executor_deleted_at:` for false checks

Replace with `not Subtask.executor_deleted_at`

(E712)